Update README.md

Namasya opened the pull request:
Resolves Issue #<issue_number>

### Description
Please explain the changes you made in detail here.

### Technical Specifications

### Screenshots (if required)

### How to run
Please list all the steps required to run in details. Preferably specify steps for all platforms.

### Checklist
- [ ] Code compiles correctly.
- [ ] Changes are tested properly.
- [ ] Added the README / documentation, if necessary.
- [ ] Followed the Contributing guidelines.
- [ ] Resolved merge conflicts, if any.

<!-- Delete all the unnecessary / inapplicable text from the template -->

Namasya committed:
Update README.md

Ram-Aditya commented:
@Namasya Please modify your comment so that you include all details about the issue you have solved.

Namasya committed:
Update README.md

Namasya committed:
Rename algorithms/kruskal_algo/README.md to algorithms/kruskal.cpp

kruskal algorithm program is created using c++






kruskal algorithm is created using c++
